roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -0,0 +1,49 @@
+# These tasks are run when aci_mode is set
+
+- name: check aci-gw container image
+  shell: docker inspect contiv/aci-gw
+  register: docker_aci_inspect_result
+  ignore_errors: yes
```

Comment:
I think this shall have `run_once` as well.

Also I am not sure if you already explored this but having `run_once` at the include level might be cleaner Also I think this is coming mostly from the limitation of number of supported netmaster instances (And accordingly number of aci-gw instances). I am hoping we can keep this work-around limited in scope as much possible to allow easier cleanup later.